### ISSUE TITLE
Azure Container Registry taking time to show up in CLI

### ISSUE THREAD

**Issue by Kiyani1** [MicrosoftLearning/AZ-204-DevelopingSolutionsforMicrosoftAzure#388]
# Module: 05
#### Step: 05

Description of issue
If we create a container registry resource using Azure it take time to show up in Azure CLI which can take anywhere around 15 minutes to 45 minutes based on my experience. However, if we create container registry using Bash command it shows up instantly in Azure CLI, I would suggest to change the task for creation of Azure Container registry to Bash command using the following command:

az acr create --resource-group ContainerCompute \
  --name <Registry Name> --sku Basic

![image](https://user-images.githubusercontent.com/98971092/152582227-bc078f9c-4da2-446a-87bc-24341d3373da.png)

**Comment by WigF1** [comment]
I second this! Having to wait for it show up is a constant source of annoyance for students and MCTs alike.

**Comment by JeffKoMS** [comment]
The instructions shown in the image above are from before the updates last November. They are pretty much the same steps, but since I can't see all of the instructions can you confirm they are directing the student to use the Basic SKU? I've just done some testing in 2-3 regions and creating the container registries through the portal took about 5 seconds. I haven't tested it on a student pass yet, but will shortly. 

I don't disagree with the change proposed - would make the steps quite a bit shorter which is always a good thing. But, since I can't replicate the problem I can't be confident it would solve the problem which is the most important thing.

**Comment by Kiyani1** [comment]
> The instructions shown in the image above are from before the updates last November. They are pretty much the same steps, but since I can't see all of the instructions can you confirm they are directing the student to use the Basic SKU? I've just done some testing in 2-3 regions and creating the container registries through the portal took about 5 seconds. I haven't tested it on a student pass yet, but will shortly.
> 
> I don't disagree with the change proposed - would make the steps quite a bit shorter which is always a good thing. But, since I can't replicate the problem I can't be confident it would solve the problem which is the most important thing.

Yes, they are instructing to use the Basic SKU, the problem is not with creation of container registries. The issue starts on the next task where we have to get list of container registries through bash using "az acr list". It can take anywhere from around 15 to 45 minutes for container registries to show up in CLI or Bash based on my testing. If we create container registries through bash they show up immediately.

![image](https://user-images.githubusercontent.com/98971092/153336796-4cdcd457-d489-437f-ba31-5970b74de715.png)

**Comment by JeffKoMS** [comment]
I just merged #394 to address this issue.
